src/mono/wasm/debugger/BrowserDebugProxy/Firefox/FirefoxMonoProxy.cs

```diff
@@ -794,6 +794,19 @@ private static JObject ConvertToFirefoxContent(ValueOrError<GetMembersResult> re
         return variables;
     }
 
+    protected override async Task<bool> IsRuntimeAlreadyReadyAlready(SessionId sessionId, CancellationToken token)
+    {
+        if (contexts.TryGetValue(sessionId, out ExecutionContext context) && context.IsRuntimeReady)
+            return true;
+
+        Result res = await SendMonoCommand(sessionId, MonoCommands.IsRuntimeReady(RuntimeId), token);
+        if (!res.IsOk) //if runtime is not ready this may be the response
+            return false;
+        if (res.Value?["result"]?["value"] is JObject) //if runtime is not ready this may be the response
```

Comment:
Should we instead check for `res.Value?["result"]?["value"]?.Type == JTokenType.Boolean`?

Reply:
Can we do that in the base method itself?

Reply:
done